pbs_light.go

```diff
@@ -179,7 +179,7 @@ type cookieSyncResponse struct {
 
 func cookieSync(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	mCookieSyncMeter.Mark(1)
-	userSyncCookie := pbs.ParsePBSCookieFromRequest(r)
+	userSyncCookie := pbs.ParsePBSCookieFromRequest(r, &(hostCookieSettings.OptOutCookie))
 	if !userSyncCookie.AllowSyncs() {
```

Comment:
Just double-checking... in your previous diff, there was a line here: `userSyncCookie.SetCookieOnResponse(w, hostCookieSettings.Domain)`

Did you mean to remove that in this one?

Reply:
Yes, this was to allow using a cookie that was already set. PBS reads the cookie when it can, with every request, so there's no reason to set the cookie on the PBS domain again

Reply:
righto... that makes sense